suite/performance/vrf_test.go

```diff
@@ -0,0 +1,82 @@
+package performance
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/smartcontractkit/integrations-framework/actions"
+	"github.com/smartcontractkit/integrations-framework/client"
+	"github.com/smartcontractkit/integrations-framework/environment"
+	"github.com/smartcontractkit/integrations-framework/tools"
+	"math/big"
+	"time"
+)
+
+var _ = Describe("VRF soak test @soak-vrf", func() {
+	var (
+		s        *actions.DefaultSuiteSetup
```

Comment:
change `s` to something like `suiteSetup`, makes it more explicit what it actually is referring to